templates/form/dsfr_theme.html.twig

```diff
@@ -127,11 +127,12 @@
 
 {# custom type #}
 {% block search_checkbox_row %}
-    <div class="search-checkbox-container {% if errors|length > 0 %}fr-select-group--error{% endif %}">   
+    <div class="search-checkbox-container fr-input-group {% if errors|length > 0 %}fr-select-group--error{% endif %}">   
         <div class="fr-input-group fr-mb-0">
-            <input type="text" placeholder="{{noselectionlabel}}" class="fr-input">
+            <label for="{{ id }}" class="fr-label">{{ label }}</label>
```

Comment:
Plutôt utiliser le `{{block('form_label')}}`, merci

Reply:
vu ensemble : nécessaire pour l'attribut `for` ; ajout d'un contrôle sur l'existance du label